src/Core/Usecase/Form/VerifyFormLoaded.php

```diff
@@ -45,7 +45,10 @@ protected function verifyFormExists()
 	// Usecase
 	public function interact()
 	{
-		$this->verifyFormExists();
+		if ($this->getRequiredIdentifier('form_id')) {
```

Comment:
This trait is used in many places: i.e create / edit / delete form stages .. the form_id should only be optional for SearchFormStage and SearchFormAttribute Usecases. Not sure what the best way to handle that is? Maybe just override `verifyFormExists` on those Usecases instead?